### PR TITLE
Implement carousel in new homepage

### PR DESCRIPTION
**Pull request by suecarmol**
## Description
Implement filterable carousel of Wikipedia Library collections.

## Rationale
[//]: # (Why is this change required? What problem does it solve?)
This is part of the new Wikipedia Library redesign.

## Phabricator Ticket
[//]: # (Link to the Phabricator ticket)
[T282114](https://phabricator.wikimedia.org/T282114)

## How Has This Been Tested?
[//]: # (- Did you add tests to your changes? Did you modify tests to accommodate your changes?)
[//]: # (- Can this change be tested manually? How?)
Manually tested the carousel in different resolutions and in RTL and LTR languages.

## Screenshots of your changes (if appropriate):
[//]: # (It can also be a GIF to prove that your changes are working)
Screen Shot 2021-06-29 at 19 20 58

Screen Shot 2021-06-29 at 19 21 11

**Tablet view**
Screen Shot 2021-06-29 at 19 21 24

## Types of changes
What types of changes does your code introduce? Add an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Minor change (fix a typo, add a translation tag, add section to README, etc.)

### CODE REVIEW

File: TWLight/static/css/new-local.css [hunk 1]
```diff
@@ -189,6 +191,298 @@ LOGIN PARTIAL CSS
...
+}
+.partner-image-description-container{
+    position: relative;
+    height: 176px;
+    margin-left: 25px;
+    margin-right: 30px;
+}
+.partner-glider-image{
+    max-width: 100%;
+    height: 125px;
```
Comment (jsnshrmn) [inline TWLight/static/css/new-local.css:411]: Must have overlooked this in my image scaling work. This change resolves the only remaining partner logo stretching.
```suggestion
    max-height: 125px;
```

File: TWLight/static/css/new-local.rtl.css [hunk 2]
```diff
@@ -189,6 +189,292 @@ LOGIN PARTIAL CSS
     padding: 0px 10px 0px 0px;
 }
 
+/*
+----------------------------------------------------------------------------
+HOMEPAGE CAROUSEL CSS
+--------------------------------------------------------------------------
+*/
```
Comment (jsnshrmn) [inline TWLight/static/css/new-local.rtl.css:196]: This file seems out of date. Can we just delete and regenerate with our cssjanus script?

**Review by jsnshrmn — CHANGES_REQUESTED** [review]
This is looking really good. I love how clean and well-commented your code is here.
I have one small inline change request and one question, but this is otherwise mergable IMO

File: TWLight/static/css/new-local.css (hunk 1 above)
Comment (suecarmol) [reply TWLight/static/css/new-local.css]: Done!

File: TWLight/static/css/new-local.rtl.css (hunk 2 above)
Comment (suecarmol) [reply TWLight/static/css/new-local.rtl.css:196]: I will make the changes manually.

Comment (suecarmol) [reply TWLight/static/css/new-local.rtl.css:196]: This class has been updated! Let me know if I missed something!

**Review by jsnshrmn — APPROVED** [review]

**Pull request merged**